### ISSUE TITLE
Default config file does not work

### ISSUE THREAD

**Issue by noteven** [captainhookphp/captainhook#242]
Hello, I am unsure if this might be an expected behaviour, but the default config file generated by `vendor/bin/captainhook configure` does not work out of the box as it uses `CaptainHook\\Phar\\CaptainHook\\App\\Hook\\Message\\Action\\Beams` - changing this manually to `\\CaptainHook\\Phar\\CaptainHook\\App\\Hook\\Message\\Action\\Beams` corrected the issue.  I am not sure if it pertinent but I am running CaptainHook inside of a docker container.

 The error message I received before making the necessary correction was:
```
pre-commit: 
 - CaptainHook\Phar\CaptainHook\App\Hook\PHP\Action\Linting          : failed
captainhook failed executing all actions, took: 0.00s

CaptainHook\Phar\CaptainHook\App\Hook\PHP\Action\Linting
  command output:
sh: CaptainHookPharCaptainHookAppHookPHPActionLinting: not found
failed to execute: CaptainHook\Phar\CaptainHook\App\Hook\PHP\Action\Linting
```

Generated config:
```json
{
    "commit-msg": {
        "enabled": true,
        "actions": [
            {
                "action": "CaptainHook\\Phar\\CaptainHook\\App\\Hook\\Message\\Action\\Beams",
                "options": {
                    "subjectLength": 50,
                    "bodyLineLength": 72
                }
            }
        ]
    },
    "pre-push": {
        "enabled": false,
        "actions": []
    },
    "pre-commit": {
        "enabled": true,
        "actions": [
            {
                "action": "CaptainHook\\Phar\\CaptainHook\\App\\Hook\\PHP\\Action\\Linting"
            },
            {
                "action": "phpunit"
            }
        ]
    },
    "prepare-commit-msg": {
        "enabled": false,
        "actions": []
    },
    "post-commit": {
        "enabled": false,
        "actions": []
    },
    "post-merge": {
        "enabled": false,
        "actions": []
    },
    "post-checkout": {
        "enabled": false,
        "actions": []
    },
    "post-rewrite": {
        "enabled": false,
        "actions": []
    },
    "post-change": {
        "enabled": false,
        "actions": []
    }
}
```

**Comment by sebastianfeldmann** [comment]
I just confirmed the issue, I'll investigate and fix it ASAP

**Comment by sebastianfeldmann** [comment]
I fixed the wrongful path replacement

The correct value should be `\\CaptainHook\\App\\Hook\\Message\\Action\\Beams` or `\\CaptainHook\\App\\Hook\\PHP\\Action\\Linting`

Should be fixed in version `5.2.22`

Thanks for reporting the problem, if you still have problems just reopen this issue.
